route/oohdata-route.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+const storage = require('../lib/storage.js');
+const response = require('../lib/response.js');
+const Oohdata = require('../model/oohdata.js');
+
+module.exports = function(router) {
+  router.get('/api/oohdata', function(req, res) {
+
+    if (req.url.query.id) {
+      storage.fetchItem('oohdata', req.url.query.id)
+      .then( oohdata => {
+
+        response.sendJSON(res, 200, oohdata);
+      })
+      .catch( err => {
```

Comment:
you are passing `err` as a param to the catch block which is correct but then you don't do anything with it. You should use a `console.error(err);` to print the error to the console.